pkgs/applications/audio/bespokesynth/default.nix

```
@@ -50,64 +29,57 @@ let
 in
 stdenv.mkDerivation rec {
   pname = "bespokesynth";
-  version = "1.0.0";
+  version = "1.0.0-4a74e80";
```

https://nixos.org/manual/nixpkgs/stable/#sec-package-naming

> - If a package is not a release but a commit from a repository, then the version part of the name *must* be the date of that (fetched) commit. The date *must* be in `"YYYY-MM-DD"` format. Also append `"unstable"` to the name - e.g., `"pkgname-unstable-2014-09-23"`.

```suggestion
  version = "unstable-2021-11-03";
```

\+ change commit & PR titles accordingly.